src/SilverStripe/BehatExtension/Context/BasicContext.php

```diff
@@ -843,7 +843,7 @@ public function theTextBeforeAfter($textBefore, $order, $textAfter, $element) {
 	public function iWaitXUntilISee($wait, $selector) {
 		$page = $this->getSession()->getPage();
 
-		$this->spin(function($page) use ($page, $selector){
```

Comment:
Error was ambiguous value of $page due to it both inheriting from parent context as well as being an arg.

Reply:
Yep, that looks dubious and removing it makes sense in the context of these step definitions